### PR TITLE
Improved design

### PR DESCRIPTION
**Pull request by juhoojala**
This PR adapts the UI to the new design.

Responsiviness will be done in another PR.

### CODE REVIEW

**Comment by vsaarinen** [comment]
Very nice! After a very quick testing, ran into these issues:

screenshot 2017-05-04 22 25 29
If a description is missing, the project cards become smaller (vertically). Is this on purpose?

screenshot 2017-05-04 22 25 41
This is an old bug, but in case you can get it fixed now: with a really long word, the project cards become wider than they should.

The "Latest preview" links in the project cards don't work.

**Comment by vsaarinen** [comment]
Should the Minard logo in the top left corner be a link to the Team Projects View?

**Comment by vsaarinen** [comment]
- I was expecting the comments in the activity feed to be links to the comment in the Deployment View. Not sure about the commit text in the activity feed.
- The screenshots in the activity feed look pretty broken, especially with the hover effect: 
screenshot 2017-05-04 22 33 15

File: src/js/components/_mixins.scss [hunk 1]
```diff
@@ -23,16 +18,27 @@
...
+}
+
 @mixin font-size-XS {
   font-size: 0.563rem;
 }
 
 @mixin font-size-S {
-  font-size: 0.796rem;
+  // font-size: 0.796rem;
+  font-size: 0.875rem;
```
Comment (vsaarinen) [inline src/js/components/_mixins.scss]: Is this XS size still used for something or should it be removed?

**Comment by vsaarinen** [comment]
Should there be a hover effect for the links in e.g. the activity feed?

File: src/js/components/_mixins.scss (hunk 1 above)
Comment (juhoojala) [inline src/js/components/_mixins.scss]: Removed

**Comment by juhoojala** [comment]
Added hover effect and fixed the other issues, except the screenshot

File: src/js/components/common/activity-list/activity-group.tsx [hunk 2]
```diff
@@ -7,10 +7,12 @@ import TimeAgo from 'react-timeago';
 import { Activity, ActivityType } from '../../../modules/activities';
 import { isSuccessful } from '../../../modules/deployments';
 
+import Avatar from '../../common/avatar';
+import BuildStatus from '../../common/build-status';
```
Comment (vsaarinen) [inline src/js/components/common/activity-list/activity-group.tsx]: Don't need `../common/` for this and the one above.

File: src/js/components/common/simple-section-title.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+
+const styles = require('./simple-section-title.scss');
+
+// NOTE: Each of these are expected to have one root element.
+// CSS classes will be added to this element.
```
Comment (vsaarinen) [inline src/js/components/common/simple-section-title.tsx]: This comment probably doesn't apply?

File: src/js/components/project-view/branch-summary.tsx [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
 import { connect } from 'react-redux';
+import TimeAgo from 'react-timeago';
+import Icon = require('react-fontawesome');
 
 import { Branch } from '../../modules/branches';
 import Commits, { Commit } from '../../modules/commits';
 import Deployments, { Deployment } from '../../modules/deployments';
 import { FetchError, isFetchError } from '../../modules/errors';
 import { StateTree } from '../../reducers';
+import Avatar from '../common/avatar';
```
Comment (vsaarinen) [inline src/js/components/project-view/branch-summary.tsx]: Should be in the import block below (for components).

File: src/js/components/project-view/project-branches.tsx [hunk 5]
```diff
@@ -8,7 +8,7 @@ import { Project } from '../../modules/projects';
 
 import LoadingIcon from '../common/loading-icon';
 import MinardLink from '../common/minard-link';
-import SectionTitle from '../common/section-title';
+import SectionTitle from '../common/simple-section-title';
```
Comment (vsaarinen) [inline src/js/components/project-view/project-branches.tsx]: Should be SimpleSectionTitle

File: src/js/components/sub-header.scss [hunk 6]
```diff
@@ -25,8 +25,8 @@
 }
 
 .sub-header-link {
-  @include nitti-grotesk-medium;
-  @include font-size-MS;
+  // @include nitti-grotesk-medium;
```
Comment (vsaarinen) [inline src/js/components/sub-header.scss]: Comments

File: src/js/components/sub-header.tsx [hunk 7]
```diff
@@ -1,13 +1,15 @@
...
+import { connect, Dispatch } from 'react-redux';
+import Icon = require('react-fontawesome');
 
 import { FetchError, isFetchError } from '../modules/errors';
+import Modal, { ModalType } from '../modules/modal';
 import Projects, { Project } from '../modules/projects';
 import Selected from '../modules/selected';
 import User, { Team } from '../modules/user';
 import { StateTree } from '../reducers';
+import Avatar from './common/avatar';
```
Comment (vsaarinen) [inline src/js/components/sub-header.tsx]: Components should be in their own import group. Looks like this was not done correctly previously.

File: src/js/components/sub-header.tsx [hunk 8]
```diff
@@ -24,48 +26,106 @@ interface GeneratedProps {
...
-    // TODO: what if we don't have team?
+type Props = GeneratedProps & GeneratedDispatchProps;
+
+class SubHeader extends React.Component<Props, void> {
+
+  private getLeftContent() {
+    const { openPageType, project, branch, team } = this.props;
+    const items: JSX.Element[] = [];
+
+    items.push(
```
Comment (vsaarinen) [inline src/js/components/sub-header.tsx]: The array could be initialized with this value.

File: src/js/components/team-projects-view/activity-section.tsx [hunk 9]
```diff
@@ -3,7 +3,7 @@ import * as React from 'react';
 import { Activity } from '../../modules/activities';
 
 import ActivityList from '../common/activity-list';
-import SectionTitle from '../common/section-title';
+import SectionTitle from '../common/simple-section-title';
```
Comment (vsaarinen) [inline src/js/components/team-projects-view/activity-section.tsx]: Should be SimpleSectionTitle.

File: src/js/components/team-projects-view/project-card.scss [hunk 10]
```diff
@@ -39,25 +36,25 @@
 }
 
 .card-middle {
-  padding: 36px;
-  height: 178px;
+  padding-left: 20px;
```
Comment (vsaarinen) [inline src/js/components/team-projects-view/project-card.scss]: Maybe just `padding: 20px;`?

File: src/js/components/team-projects-view/project-card.scss [hunk 11]
```diff
@@ -39,25 +36,25 @@
...
+  padding-right: 20px;
+  padding-bottom: 20px;
   background-color: white;
-  border-top: 1px solid $minard-lightest-gray;
-
   color: $minard-mid-gray;
   @include font-size-MS;
-  letter-spacing: -0.11px;
+  // letter-spacing: -0.11px;
```
Comment (vsaarinen) [inline src/js/components/team-projects-view/project-card.scss]: Comment

File: src/js/components/team-projects-view/project-card.tsx [hunk 12]
```diff
@@ -85,32 +116,33 @@ const ProjectCard = ({ project, latestDeployedCommit, latestDeployment }: Passed
...
-            <Truncate lines={3}>
-              {project.description}
-            </Truncate>
-          </p>
+          <MinardLink project={project}>
+            <h3 className={styles.title}>
+              {project.name}
+            </h3>
+          </MinardLink>
+          { latestDeployment ? (
```
Comment (vsaarinen) [inline src/js/components/team-projects-view/project-card.tsx]: No space after `{`

File: src/js/components/team-projects-view/projects-section.tsx [hunk 13]
```diff
@@ -12,7 +12,7 @@ import { StateTree } from '../../reducers';
 
 import LoadingIcon from '../common/loading-icon';
 import MinardLink from '../common/minard-link';
-import SectionTitle from '../common/section-title';
+import SectionTitle from '../common/simple-section-title';
```
Comment (vsaarinen) [inline src/js/components/team-projects-view/projects-section.tsx]: Should be SimpleSectionTitle

File: src/js/components/team-projects-view/projects-section.tsx [hunk 14]
```diff
@@ -65,13 +75,30 @@ class ProjectsSection extends React.Component<Props, void> {
...
-            {showAll ? 'All' : 'Latest'} projects for <span className={styles.team}>{team!.name}</span>
+            Projects
           </span>
         </SectionTitle>
-        <FlipMove className="row center-xs start-sm" enterAnimation="elevator" leaveAnimation="elevator">
+        <FlipMove
+          className={classNames(
+            showAll && 'row center-xs start-sm',
+            showAll && 'center-xs',
+            showAll && 'start-sm',
```
Comment (vsaarinen) [inline src/js/components/team-projects-view/projects-section.tsx]: Is this correct?

Comment (juhoojala) [inline src/js/components/team-projects-view/projects-section.tsx]: I fix this in the responsiviness PR

File: src/js/components/common/activity-list/activity-group.tsx (hunk 2 above)
Comment (juhoojala) [inline src/js/components/common/activity-list/activity-group.tsx]: Fixed

File: src/js/components/common/simple-section-title.tsx (hunk 3 above)
Comment (juhoojala) [inline src/js/components/common/simple-section-title.tsx]: Removed

File: src/js/components/project-view/project-branches.tsx (hunk 5 above)
Comment (juhoojala) [inline src/js/components/project-view/project-branches.tsx]: Fixed

**Pull request merged**